### ISSUE TITLE
When button is disabled you can still click on icon

### ISSUE THREAD

**Issue by albertb22** [Dogfalo/materialize#947]
If you have a button tag that is disabled and you have an icon in it. You can still click on the icon to get the waves effect.

**Comment by TheRinseM** [comment]
+ 1

**Comment by TheRinseM** [comment]
http://0.media.dorkly.cvcdn.com/36/35/6603dc5a9292104b44c349b85b5aaf7a-5-crazy-fan-theories-that-make-total-sense.jpg

**Comment by coltonfitzgerald** [comment]
Under what circumstances? I'm trying to recreate the issue but I can't seem to. Can you post the HTML markup that it doesn't work with?

**Comment by TheRinseM** [comment]
@coltonfitzgerald The code we used:
``` 
<div class="input-field right-align right">
      <button id="submitLoginForm" class="btn waves-effect waves-light blue darken-4">Login
              <i class="mdi-content-send right"></i>
    </button>
</div>

**Comment by lamoni** [comment]
I've submitted pull #970 as a fix for this.

**Comment by TheRinseM** [comment]
@lamoni Thank you for the fix!

**Comment by lamoni** [comment]
@TheRinseM gladly!

**Comment by coltonfitzgerald** [comment]
But here's the thing. (Correct me if I'm wrong) I thought you use the class ".disabled" to disable buttons whereas you are setting the attribute "disabled="true"" with that line of jQuery

**Comment by lamoni** [comment]
Adding class of 'disabled' to buttons actually doesn't stop the waves-effect either.  Assuming someone else doesn't do it, I'll add that check in as well.

**Comment by albertb22** [comment]
Thank you for the fix.

**Comment by Dogfalo** [comment]
Fixed in 345af00
